omigami/spec2vec.py

```diff
@@ -20,14 +20,17 @@ class InvalidCredentials(Exception):
 
 
 class Spec2Vec:
-    _endpoint_url = (
+    _PREDICT_ENDPOINT_POSITIVE_ION_MODE = (
         "https://omigami.datarevenue.com/seldon/seldon/spec2vec/api/v0.1/predictions"
     )
+    _PREDICT_ENDPOINT_NEGATIVE_ION_MODE = "https://omigami.datarevenue.com/seldon/seldon/spec2vec/api/v0.1/predictions_negative"
 
     def __init__(self, token: str):
         self._token = token
 
-    def match_spectra_from_path(self, mgf_path: str, n_best: int) -> List[pd.DataFrame]:
+    def match_spectra_from_path(
+        self, mgf_path: str, n_best: int, negative_ion_mode: bool = False
+    ) -> List[pd.DataFrame]:
```

Comment:
I prefer if we should have a str input parameter called ion_mode and not a boolean flag. This could be used to change the endpoint name. also the endpoints will be named positive/negative, not predictions and predictions_negative

Reply:
Ok, can change that